src/scripts/farming/mutation/mutationTypes/FieldMutation.ts

```diff
@@ -44,7 +45,7 @@ class FieldMutation extends GrowMutation {
      */
     get unlocked(): boolean {
         // Check for Berry requirements
-        if (!App.game.farming.unlockedBerries[this.fieldBerry]()) {
+        if (this.fieldBerries.some((fb) => !App.game.farming.unlockedBerries[fb.berry]())) {
```

Comment:
Why only some and not every?

Reply:
If any if the required berries are not unlocked, this berry cannot be unlocked. That is how it should work, right?

Reply:
yeah, I misread the code. seems fine